src/main/java/org/cactoos/text/NormalizedText.java

```diff
@@ -51,17 +47,13 @@ public NormalizedText(final String text) {
      * @param text A Text
      */
     public NormalizedText(final Text text) {
-        this.origin = text;
+        super(
+            (Scalar<String>) () -> new ReplacedText(
+                new TrimmedText(text),
+                "\\s+",
+                " "
+            ).asString()
+        );
```

Comment:
@krzyk Same thing here, our objects should be lazy to apply these transformations to our text objects and do this outside the constructor.